source/unified-test-format/unified-test-format.rst

```diff
@@ -1143,8 +1145,8 @@ The structure of each object is as follows:
   to be observed. See `commonOptions_client`_.
 
 - ``eventType``: Optional string. Specifies the type of the monitor which
-  captured the events. Valid values are ``command`` for `Command Monitoring
-  <../command-monitoring/command-monitoring.rst#api>`__ events, ``cmap`` for
+  captured the events. Valid values are ``command`` for `Command Logging and Monitoring
+  <../command-logging-and-monitoring/command-logging-and-monitoring.rst#api>`__ events, ``cmap`` for
```

Comment:
the tag should now be `#events-api` like
https://github.com/mongodb/specifications/blob/master/source/command-logging-and-monitoring/command-logging-and-monitoring.rst#events-api (same for links below this)